### DISCUSSION TITLE
task icon blurry

### DISCUSSION THREAD

**Discussion by mianxiu** [webui-dev/webui#462] [Q&A]
how can i fix task icon blurry?(I use a svg to favicons icon)
![Snipaste_2024-07-29_23-53-20](https://github.com/user-attachments/assets/6b78319a-cfeb-43ad-836c-cabde143e042)

1. Window 11
2. Python 
3. WebUI v2.5 (replace dll to package)

**Comment by AlbertShown** [comment]
This issue is because of the icon itself, and not the library. To double check, just use the icon in a simple `.html` local file, then open it in your default web browser, you may see the same blurry icon.

To fix it, make the SVG icon bigger, or use the PNG format.

**Reply by mianxiu** [reply]
Thank fo reply.🫡
I took your advice and tried，and i use other program ico (like foobar), but it still not work. i found it just use 16x16px size. I try use win.set_icon to set svg file (size 512x512), aslo try set png like: 
![image](https://github.com/user-attachments/assets/c9d0d527-45ad-4007-9496-ca9752c2df03)
![image](https://github.com/user-attachments/assets/dc31a5ac-a7ef-4c39-9735-0f2760d0858d)
still not work 😢

**Reply by AlbertShown** [reply]
Put the icon in a simple html file and run it to make sure the browser is showing correctly.
